app/views/course/forum/posts/edit.html.slim

```diff
@@ -0,0 +1,3 @@
+= page_header
+
+= render partial: 'form', locals: { url: course_forum_topic_post_path(current_course, @forum, @topic, @post) }
```

Comment:
hmm...I've tried this, it will raise exception...`undefined local variable or method `url'`